news/2024/2024-02-16-osumania-7k-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,106 @@
+
+### Japan ::{ flag=JP }:: vs. ::{ flag=AU }:: Australia
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2059665757&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Japan vs. Australia was yet another match many people were looking for. There were several reasons for this, such as their similarly seeded placement or the amazing performances they've already shown throughout the tournament. Suffice to say, the anticipation was overwhelming — and it didn't disappoint!
+
+The teams started on a good foot by choosing their bans, protects, and first pick wisely. Australia had an edge on long note picks, whereas Japan has been consistently demolishing rice maps on all rounds. That's why on their first pick, they opened up with [Sydosys - Voyage](https://osu.ppy.sh/beatmapsets/2130624#mania/4480927). And ::{ flag=JP }:: [rainlily](https://osu.ppy.sh/users/485007)'s score should've been indicative of Japan's rice prowess right off the bat.
+
+The match went on with Australia picking [ARForest - The Last Page](https://osu.ppy.sh/beatmapsets/2130622#mania/4480923), securing their protect and answering to Japan's protect, and ::{ flag=AU }:: [ruka](https://osu.ppy.sh/users/6117525) showing off Australia's strong suit in long note maps.
+
+The rice vs. long note back and forth went on, with Japan picking [Junk - Kagaribi](https://osu.ppy.sh/beatmapsets/2130665#mania/4481018), Australia answering with [Getty - Revolt](https://osu.ppy.sh/beatmapsets/2130612#mania/4480900), then [RykSeb - RETINAE](https://osu.ppy.sh/beatmapsets/2130601#mania/4480882). Australia gambled their chances on [tn-shi - Synthesis](https://osu.ppy.sh/beatmapsets/2130789#mania/4481282) and managed to survive a potential break point on what ended up being the closest pick yet!
+
+Unfortunately for Australia, despite making a huge effort by keeping up with Japan, when Japan managed to get a match point at 5–4, Australia wasn't able to break through their pick, [Frums - 24eeev0-$](https://osu.ppy.sh/beatmapsets/2130620#mania/4480921), and ended up losing the break point.
+
+Despite how the match ended, it is safe to assume nobody was sure of who was going to be the winner given both teams' phenomenal performance. A nerve-wracking match full and through.
+
+---
+
+The tournament is slowly coming to a close, with 2 more weeks left ahead of us. It would be a real shame if you made it this far without checking out any of the matches live, so be sure to visit the [osulive Twitch channel](https://www.twitch.tv/osulive) this weekend for some high intensity mania action! If you are interested in more details regarding the tournament, such as the Finals pool or the full bracket — the [wiki article](/wiki/Tournaments/MWC/2024_7K) has everything you need!
```

Comment:
```suggestion
The tournament is slowly coming to a close, with 2 more weeks left ahead of us. It would be a real shame if you made it this far without checking out any of the matches live, so be sure to visit the [osulive Twitch channel](https://www.twitch.tv/osulive) this weekend for some high-intensity osu!mania action!

If you are interested in more details regarding the tournament, such as the Finals pool or the full bracket — the [wiki article](/wiki/Tournaments/MWC/2024_7K) has everything you need! (It also has the match schedules, which are updated regularly, so you probably want to check it out anyway to know when things go live!)
```